packages/react-button/src/components/MenuButton/renderMenuButton.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { getSlots } from '@fluentui/react-utilities';
+import { MenuContext } from '@fluentui/react-shared-contexts';
+import { MenuButtonState } from './MenuButton.types';
+import { menuButtonShorthandProps } from './useMenuButton';
+
+/**
+ * Redefine the render function to add slots. Reuse the button structure but add
+ * slots to children.
```

Comment:
(nit) This comment doesn't really make sense to me. Since it's the documentation for a public function, might want to clean it up?

Reply:
Done!